aten/src/ATen/native/cuda/linalg/BatchLinearAlgebraLibBlas.cpp

```diff
@@ -0,0 +1,268 @@
+#define TORCH_ASSERT_ONLY_METHOD_OPERATORS
```

Comment:
I guess this was moved from the other file so that we can exclude it for hipify? 
Can we have a comment in this file to that effect if it is true

Reply:
Done in 8fdf1544c48f80337ecbb8c38e0347278547cd98.